src/components/animated/_spotlight.less

```diff
@@ -0,0 +1,151 @@
+@import "./_variables.less";
+@colors : @blue,@pink,@red,@green,@yellow,@orange,@purple,@black;
+@colorbtns:blue-btn,pink-btn,red-btn,green-btn,yellow-btn,orange-btn,purple-btn,black-btn;
+.spotlight-colorspecific-mixin(@param) when (@param = "main")
+{
+	each(@colorbtns,{
+		&.@{value}{
+			@currentcolor:extract(@colors,@key);
+			border-color:@currentcolor;
+			box-shadow:inset 0px 0px 10px @currentcolor;	
+		}
+	})
+} 
+.spotlight-colorspecific-mixin(@param) when (@param = "after/before")
+{
+	each(@colorbtns,{
+		&.@{value}{
+			@currentcolor:extract(@colors,@key);
+			&:after{
+			border-color:@currentcolor;
+			box-shadow:inset 0px 0px 10px @currentcolor;
+			}
+			&:before{
+				background-color: extract(@colors,@key);
+				border-color:white ;
+			}	
+		}
+	})
+} 
+.sbtn.spotlight-btn{
+	position: relative;
+	padding:10px;
+	color:black;
+	height:unset;
+	width:unset;
+	background-color: transparent;
+	overflow:visible !important;
+	cursor:pointer;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	outline:none;
+	border-width: 3px; 
+	border-style: solid;
+	z-index: 0;
+	.spotlight-colorspecific-mixin("main");
+	&.block-btn{
+		height: 50px;
+		width: 200px;
+	}
+	&.white-btn{
+		border-color: #9c9a9a;
+		box-shadow: inset 0px 0px 10px  #b1afaf; 
+		&:after{
+			border-color:#9c9a9a;
+			box-shadow: inset 0px 0px 10px  #b1afaf;
+		} 
+		&:before{
+			background-color: white;
+			border:1px solid #9c9a9a;
+		}
+	} 
+	.spotlight-colorspecific-mixin("after/before");
+	&:after{             
+		content:"";
+		border-width:3px;
+		border-style: solid;
+		height:100%;
+		width:100%;
+		top:3px;
+		left:3px;
+		opacity: 1;
+		position: absolute;
+		z-index: -10;
+		background-color: transparent;
+		overflow:visible;
+		box-sizing: content-box;
+		border-radius: inherit;	
+	}
+	&:hover{
+			animation: font_color_change 0.5s ease-in 0.25s forwards !important;
+			&:before{
+				animation: spotappear 0.5s ease-in forwards, spotexpand 0.25s ease-in 0.5s forwards;
+			}
+			&:after{
+				animation: after_returns_to_main 0.5s ease-in forwards;
+			}
+	}
+	&:before{
+		content: "";
+		padding:20px;
+		position: absolute;
+		height:100%;
+		width:50px;
+		top:100px;
+		opacity:0;
+		z-index: -1;
+		left:50%;
+		transform: translateX(-50%);
+		border-radius: 50%;
+		border-radius: inherit;
```

Comment:
remove lines 100 and 101

Reply:
@mrSidSat this change hasn't been made

Reply:
If this line is removed the spotlight that appears won't be circle and the border won't be consistent over the animation

Reply:
it is removed in the latest commit